src/pages/_document.js

```
@@ -12,8 +12,20 @@ class MyDocument extends Document {
 
     render() {
         return (
-            <Html>
-                <Head />
+            <Html data-theme="light">
+                <Head>
+                    <script
+                        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)